adapter/sfml/sfml.go

```diff
@@ -0,0 +1,56 @@
+package sfml
+
+import (
+	"fmt"
+	"github.com/c3pm-labs/c3pm/config"
+	"github.com/c3pm-labs/c3pm/internal/cmake"
+	"path/filepath"
+)
+
+type Adapter struct{}
+
+func (a *Adapter) Build(pc *config.ProjectConfig) error {
+	cmakeVariables := map[string]string{
+		"CMAKE_LIBRARY_OUTPUT_DIRECTORY":         pc.ProjectRoot,
+		"CMAKE_LIBRARY_OUTPUT_DIRECTORY_RELEASE": pc.ProjectRoot,
+		"CMAKE_ARCHIVE_OUTPUT_DIRECTORY":         pc.ProjectRoot,
+		"CMAKE_ARCHIVE_OUTPUT_DIRECTORY_RELEASE": pc.ProjectRoot,
+		"CMAKE_RUNTIME_OUTPUT_DIRECTORY":         pc.ProjectRoot,
+		"CMAKE_RUNTIME_OUTPUT_DIRECTORY_RELEASE": pc.ProjectRoot,
+		"CMAKE_BUILD_TYPE":                       "Release",
+		"BUILD_SHARED_LIB":                       "OFF",
```

Comment:
```suggestion
		"BUILD_SHARED_LIBS":                       "OFF",
```

'S' is missing, as seen in https://www.sfml-dev.org/tutorials/2.5/compile-with-cmake.php